packages/location-field/src/index.tsx

```diff
@@ -54,7 +54,9 @@ const LocationField = ({
   hideExistingValue = false,
   initialSearchResults = null,
   inputPlaceholder = null,
+  isRequired = false,
   isStatic = false,
+  isValid = true,
```

Comment:
Just so I'm clear, is this just setting up the ability to use `aria-invalid` for a future PR, not necessarily in this one?

Reply:
Yes, that will be set by OTP-RR in an upcoming PR.